### PR TITLE
chore: init gitbook

### PR DESCRIPTION
**Pull request by jeremybarbet**
## What

- Moving the handbook into island.is repository
- Grouping handbook + projects/libraries readmes into the same space
- Adding repository documentation for build/start/test/lint/format/schemas and external contributions

## Why

- Most of the people writing in the handbook are the same developers contributing to island.is repository
- Moving the handbook on island.is would make it easier for developers to update the documentation
- There is dozens of readmes for the projects/libraries and it’s hard to find them. Some readme are insides deep levels of folder and you don’t have a big picture of what exists
- Grouping handbook and readmes together would give more visibility to the handbook and would create a unique source of truth for all documentation floating around island.is

## How

- I wrote a small script that add the readmes from projects/libraries to the summary.md for the gitbook (it's used to create the navigation there)
- You can ignore a readme by adding a HTML comment in the first line of the readme and it won't be included inside the GitBook `<!-- gitbook-ignore -->`
- Did a bunch of formatting, copy typo fixes, created some missing readme.md in the projects/libraries, but still have TODOs because I don't have contexts on this stuff to be able to write about it.

## Where

Check out the [preview here](https://island-is.gitbook.io/handbook/v/feature%2Fgitbook).

### CODE REVIEW

File: apps/application-system/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Application system
+
+## About
+
+TODO
+
+## URLs
+
+- Dev: N/A
```
Comment (arnifreyrs) [inline apps/application-system/README.md:9]: umsoknir.dev01.devland.is

File: apps/application-system/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+- Staging: N/A
```
Comment (arnifreyrs) [inline apps/application-system/README.md:10]: umsoknir.staging01.devland.is

**Review by arnifreyrs — COMMENTED** [review]
looks great!

File: SUMMARY.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- [Skilavottorð](apps/skilavottord/README.md)
+- [Web](apps/web/README.md)
+
+## Libs
+
+- [CMS API](libs/api/domains/cms/README.md)
+- [API Mocks](libs/api/mocks/README.md)
+- [API Catalogue Elastic](libs/api-catalogue/elastic/README.md)
+- [API Catalogue Services](libs/api-catalogue/services/README.md)
+- [Core](libs/application/core/README.md)
```
[the remark this answers is not in the document]
Comment (jeremybarbet) [reply SUMMARY.md]: Yes, I like that! I will find a solution to keep the structure in the menu

[the remark this answers is not in the document]
Comment (jeremybarbet) [reply SUMMARY.md]: Done, you can check it out again 👍

**Review by gudjong — DISMISSED** [review]

**Review by arnifreyrs — APPROVED** [review]

File: apps/service-portal/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## About
+
+TODO
+
+## URLs
+
+- [Dev](https://service-portal.dev01.devland.is)
+- [Staging](https://service-portal.staging01.devland.is)
+- [Production](https://service-portal.island.is)
```
Comment (Thorrik58) [inline apps/service-portal/README.md:11]:
```suggestion
- [Production](https://beta.minarsidur.island.is)
```

**Review by Thorrik58 — APPROVED** [review]
LGTM apart from one URL that has changed with the release of service portal to prod.
Added a commit suggestion.

**Review by pshomov — APPROVED** [review]
LGTM

**Review by dabbeg — DISMISSED** [review]
Nice :+1:

File: apps/service-portal/README.md (hunk 4 above)
Comment (jeremybarbet) [reply apps/service-portal/README.md]: Thanks for that!

**Review by pshomov — DISMISSED** [review]
LGTM

**Review by Thorrik58 — DISMISSED** [review]

**Review by arnifreyrs — APPROVED** [review]

**Pull request merged**